x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/translations.ts

```diff
@@ -144,11 +134,23 @@ export const LOGS_SAMPLE_ERROR = {
       defaultMessage: 'The logs sample file contains non-object entries',
     }
   ),
+  LOGS_SAMPLE_FILE_TOO_LARGE: i18n.translate(
+    'xpack.integrationAssistant.step.dataStream.logsSample.fileTooLarge',
+    {
+      defaultMessage: 'Logs Sample file is too large. Please upload a file smaller than 9MB.',
```

Comment:
I would not focus on changing this size atm , since this is the maximum we can go for in an installation with 1Gig Kibana RAM.

Also, from product POV , we should not need more than ~50K events [ roughly for 9 Megs ] for generating the integration.

Reply:
This will be handled separately